packages/e2e-test-utils/src/press-key-with-modifier.js

```diff
@@ -16,12 +88,15 @@ import { modifiers, SHIFT, ALT, CTRL } from '@wordpress/keycodes';
  * @param {string} key Key to press while modifier held.
  */
 export async function pressKeyWithModifier( modifier, key ) {
-	const isAppleOS = () => process.platform === 'darwin';
+	if ( modifier.toLowerCase() === 'primary' && key.toLowerCase() === 'a' ) {
+		return await emulateSelectAll();
```

Comment:
Should this emulation be limited to macOS?

Reply:
> Should this emulation be limited to macOS?

I'd thought about it, and while true that it doesn't _need_ to occur outside macOS, I'd thought to avoid divergent implementations. The main downside is that, at least currently, `emulateSelectAll` has to do its own platform detection to determine Meta vs. Control as modifier, which wouldn't be necessary if only applying the emulation specific to macOS.